src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Todo.css'
+
+const Todo = (props) => {
+
+  return (
+    <li>
+      {props.value}
+       <input type = 'checkbox' value  = 'completed'/>
```

Comment:
You don't need to space this out. Type could be written as such: `type='checkbox'`